lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1,12 +1,11 @@
 defmodule Kernel.SpecialForms do
   @moduledoc """
-  In this module we define Elixir special forms. Special forms
-  cannot be overridden by the developer and are the basic
+  Special forms cannot be overridden by the developer and are the basic
```

Comment:
I would invert the order of the sentence "Special forms are the basic building blocks of Elixir and ..."